ExampleMod/RecipeHelper.cs

```diff
@@ -6,7 +6,37 @@ namespace ExampleMod
 {
 	public static class RecipeHelper
 	{
-		public static void AddBossRecipes(Mod mod)
+		private static readonly Mod mod = ExampleMod.instance;
```

Comment:
This line doesn't work. With a reload, you are referencing an only instance of ExampleMod, causing the recipes to fail because the items do not exist.